### ISSUE TITLE
[BUG] [Formatter] If / Endif not correctly aligned

### ISSUE THREAD

**Issue by mvsrgc** [Riverside-Healthcare/djLint#415]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: e.g. MacOS 12.6
 - Python Version 3.10.6
 - djLint Version 1.18
 - template language: jinja2

## Issue
The {% endif form %} is not correctly aligned with the {% if form %} tag. The command I am running is:
```sh
djlint . --reformat -e html.j2
```

This is what happens when I use the VS Code "Format Code" shortcut:
https://user-images.githubusercontent.com/14826113/193607175-0caaaa77-4bf0-4e96-b5e1-e261c7b50d4d.mov

## How To Reproduce
This is the contents of the html.j2 file.
```html
{% load widget_tweaks %}
{% if form %}
    <div hx-target="this" hx-swap="outerHTML">
        <form method="post">
            {% csrf_token %}
            <div class="form-row">
                <!-- Region Select -->
                <div class="form-group col-md-6 mb-3">
                    {{ form.region.label_tag }}
                    {% if form.region.errors %}
                        {% render_field form.region class+="form-control selectpicker is-invalid" data-live-search="True" data-style="btn-light" %}
                    {% else %}
                        {% render_field form.region class+="form-control selectpicker" data-live-search="True" data-style="btn-light" hx-post="/hypervisors-v2/" hx-target='#id_hypervisors' hx-swap="innerHTML" hx-trigger="change" %}
                    {% endif %}
                    {% for error in form.region.errors %}
                        <div class="text-danger">
                            <small>{{ error }}</small>
                        </div>
                    {% endfor %}
                </div>
                <!-- Hypervisor Select -->
                <div class="form-group col-md-6 mb-3">
                    {{ form.hypervisors.label_tag }}
                    {% if form.hypervisors.errors %}
                        {% render_field form.hypervisors class+="form-control selectpicker is-invalid" multiple=True data-live-search="True" data-style="btn-light" %}
                    {% else %}
                        {% render_field form.hypervisors class+="form-control selectpicker" multiple=True data-live-search="True" data-style="btn-light" %}
                    {% endif %}
                    {% for error in form.hypervisors.errors %}
                        <div class="text-danger">
                            <small>{{ error }}</small>
                        </div>
                    {% endfor %}
                </div>
                <!-- DatePicker -->
                <div class="form-group col-md-6 mb-3">
                    {{ form.date.label_tag }}
                    <div class="input-group"
                         id="{{ form.date.id_for_label }}"
                         data-target-input="nearest"
                         data-target-toggle="nearest">
                        {% if form.date.errors %}
                            {% render_field form.date class+="form-control is-invalid" %}
                        {% else %}
                            {% render_field form.date class+="form-control" %}
                        {% endif %}
                        <span class="input-group-text {% if form.date.errors %}border-danger{% endif %}"
                              data-target="#{{ form.date.id_for_label }}"
                              data-toggle="{{ form.date.id_for_label }}">
                            <i class="fas fa-calendar"></i>
                        </span>
                    </div>
                    {% for error in form.date.errors %}
                        <div class="text-danger">
                            <small>{{ error }}</small>
                        </div>
                    {% endfor %}
                </div>
                <!-- Template Select -->
                <div class="form-group col-md-6 mb-3">
                    {{ form.templates.label_tag }}
                    {% if form.templates.errors %}
                        {% render_field form.templates class+="form-control selectpicker is-invalid" data-style="btn-light" %}
                    {% else %}
                        {% render_field form.templates class+="form-control selectpicker" data-style="btn-light" %}
                    {% endif %}
                    {% for error in form.templates.errors %}
                        <div class="text-danger">
                            <small>{{ error }}</small>
                        </div>
                    {% endfor %}
                </div>
            </div>
            <button type="submit" class="btn btn-primary" hx-post=".">Submit</button>
        </form>
    {% endif form %} <------ ISSUE IS HERE
</div>

**Comment by christopherpickering** [comment]
@mvsrgc it looks like your last 2 tags are transposed.

Your `</div>` should be before the `{% endif form %}` to make them close in the same order you've opened them.

**Comment by christopherpickering** [comment]
I'm going to close this out as it doesn't appear to be a djlint issue. Please feel free to report back or open another issue as needed.
